### PR TITLE
Armenian translation short names

### PR DESCRIPTION
**Pull request by tigranbalayan**
Corrected armenian translation short names.

### CODE REVIEW

File: js/locales/bootstrap-datepicker.hy.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
 /**
  * Armenian translation for bootstrap-datepicker
- * Hayk Chamyan <hamshen@gmail.com>
+ * Tigran Balayan <balayan.tigran@gmail.com>
```
Comment (acrobat) [inline js/locales/bootstrap-datepicker.hy.js]: Don't remove the previous contributor, but add your name under it

**Comment by acrobat** [comment]
Please only keep the specific changes and leave other lines like they were. Now all lines show as diff but only a few lines do really have changes. (also you deleted the format option)

File: js/locales/bootstrap-datepicker.hy.js (hunk 1 above)
Comment (tigranbalayan) [inline js/locales/bootstrap-datepicker.hy.js]: Ok now.

**Comment by tigranbalayan** [comment]
May be it codding style, In my code tabs, in old code are spaces.

**Comment by tigranbalayan** [comment]
Should I change tab to a space?

**Comment by acrobat** [comment]
Yes please don't change the indentation, that way we only see a diff of exactly the changes you made (few string changes)

**Comment by tigranbalayan** [comment]
OK

**Comment by tigranbalayan** [comment]
Thanks for remark.

**Comment by acrobat** [comment]
Thanks @tigranbalayan

**Comment by tigranbalayan** [comment]
Thanks You too @acrobat

**Pull request closed without merging**